### ISSUE TITLE
Move default configuration settings to the app

### ISSUE THREAD

**Issue by evan-forbes** [celestiaorg/cosmos-sdk#151]
Currently we are overriding a few default configurations during the init command. We should move this to the app so that the settings we change are more explicit and don't require changes to the sdk.

**Comment by liamsi** [comment]
Yeah, I was also confused about this. Either these default param changes can be made directly in core, or, in app. App is the best if feasible 👍🏼

**Comment by rootulp** [comment]
I think most of the default overrides have been moved to celestia-app's [default_overrides.go](https://github.com/celestiaorg/celestia-app/blob/fc03393dce943532c618da53069f9331fc575b1d/app/default_overrides.go). I think the remaining revertable overrides that can be moved to celestia-app are documented [here](https://github.com/celestiaorg/cosmos-sdk/blob/2a7b89a70adefaefb0d66f71c507304ee44e8344/README.md?plain=1#L26-L34).
